commitizen/cz/base.py

```diff
@@ -74,6 +76,47 @@ def schema_pattern(self) -> Optional[str]:
         """Regex matching the schema used for message validation."""
         raise NotImplementedError("Not Implemented yet")
 
+    def validate_commit_message(
+        self, commit: git.GitCommit, pattern: str, allow_abort: bool
+    ) -> bool:
+        if not commit.message:
+            return allow_abort
+        if (
```

Comment:
Not directly relate to this PR, I think this might be something we can refactor in the future 🤔